src/main/java/JigaChat.java

```diff
@@ -0,0 +1,153 @@
+import java.util.Scanner;
+public class JigaChat {
+    Task[] taskList = new Task[100];
+    int taskCounter = 0;
+
+    public void printCommandList() {
+        printTodoCommand();
+        printDeadlineCommand();
+        printEventCommand();
+        printListCommand();
+        printMarkCommand();
+        printUnmarkCommand();
+    }
+
+    private static void printUnmarkCommand() {
+        System.out.println("unmark [task index]                         Marks a task in your list as not done");
+    }
+
+    private static void printMarkCommand() {
+        System.out.println("mark [task index]                           Marks a task in your list as done");
+    }
+
+    private static void printListCommand() {
+        System.out.println("list                                        Displays a list of all tasks and their statuses as well as types");
+    }
+
+    private static void printEventCommand() {
+        System.out.println("event [description] /from [from] /to [to]   Adds a new event to your list");
+    }
+
+    private static void printDeadlineCommand() {
+        System.out.println("deadline [description] /by [by]             Adds a new deadline to your list");
+    }
+
+    private static void printTodoCommand() {
+        System.out.println("todo [description]                          Adds a new todo to your list");
+    }
+
+    public void readCommand() {
+        String input;
+        String[] commands = new String[2];
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+        commands = input.split(" ", 2);
+
+        switch(commands[0]) {
+        case "help":
+            printCommandList();
+            return;
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            System.exit(0);
+        case "list":
+            printList();
+            return;
+        case "mark":
+            try {
+                taskList[Integer.parseInt(commands[1]) - 1].markAsDone();
+                System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as done!");
+            }
+            catch(NullPointerException e) {
+                System.out.println(commands[1] + " is not in your list!");
+            }
+            catch(ArrayIndexOutOfBoundsException e) {
+                printMarkCommand();
+            }
+            return;
+        case "unmark":
+            try {
+                taskList[Integer.parseInt(commands[1]) - 1].markAsUndone();
+                System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as not done!");
+            }
+            catch(NullPointerException e) {
+                System.out.println(commands[1] + " is not in your list!");
+            }
+            catch(ArrayIndexOutOfBoundsException e) {
+                printUnmarkCommand();
+            }
```

Comment:
try catch blocks should be of the form:

```java
try {
    // ...
} catch {
    // ...
}
```

https://se-education.org/guides/conventions/java/basic.html#layout